### DISCUSSION TITLE
[Icon Request] Accessibility Icons?

### DISCUSSION THREAD

**Discussion by ashleyisles** [tailwindlabs/heroicons#473] [Ideas]
For example - a wheelchair icon, or closed caption icon.

**Comment by josephshepin** [comment]
yes we need this

**Comment by pkrstic** [comment]
We need a copy of the Unicode Character for wheelchair ♿ (U+267F)
There is the same in Bootstrap icons https://icons.getbootstrap.com/icons/person-wheelchair/
